### PR TITLE
Edit event modal with form

### PR DESCRIPTION
**Pull request by Andreslw**
new modal added to event items

### CODE REVIEW

File: src/components/event/Create.tsx [hunk 1]
```diff
@@ -1,158 +1,177 @@
...
+const Input = styled.input({
+  borderRadius: "8px 8px 8px 8px",
+  border: "1px solid #eee",
+  marginBottom: "15px",
+  width: "88%",
+  height: "40px",
+  float: "left",
+  padding: "0px 15px"
+});
+const Input2 = styledCComponents.input``;
```
Comment (mishelashala) [inline src/components/event/Create.tsx]: delete unused variable

File: src/components/event/Create.tsx [hunk 2]
```diff
@@ -1,158 +1,177 @@
...
-    const DivFG = styled.div({
-      overflow: "hidden",
-      clear: "both"
-    })
+  createEvent = (e: any) => e.preventDefault();
 
+  render() {
     return (
       <div>
         <Form onSubmit={this.createEvent}>
```
Comment (mishelashala) [inline src/components/event/Create.tsx]: rename `createEvent` to `handleSubmit`

File: src/components/event/Create.tsx [hunk 3]
```diff
@@ -1,158 +1,177 @@
...
+      endHour: "",
+      tortaPocchuc: 0,
+      tortaCamaron: 0
     }
   };
 
-  public handleChange = (e:any) => {
-    const target = e.target;
-    const value = target.type === 'checkbox' ? target.checked : target.value;
+  handleChange = (e: any) => {
```
Comment (mishelashala) [inline src/components/event/Create.tsx]: create handle per `input`

File: src/components/event/Create.tsx [hunk 4]
```diff
@@ -1,158 +1,177 @@
...
 
             <DivLeftC>
-
               <DivFG>
                 <p>Start Event:</p>
-                <Input type="date" name="startDate" value={this.state.event!.startDate} onChange={this.handleChange}  />
+                <Input
+                  type="date"
+                  name="startDate"
+                  value={this.state.event!.startDate}
```
Comment (mishelashala) [inline src/components/event/Create.tsx]: what is the `!`?

**Review by mishelashala — CHANGES_REQUESTED** [review]

File: src/components/event/Create.tsx (hunk 4 above)
Comment (Andreslw) [reply src/components/event/Create.tsx]: I don't know it was already there

**Pull request merged**